File: iOS Snack Overflow/Snack Overflow/Pickup.swift
//
//  Pickup.swift
//  Snack Overflow
//
//  Created by MBPR on 4/18/17.
//  Copyright © 2017 Capstone. All rights reserved.
//

import Foundation

/// Eric Walton
/// 2017/04/23
/// Description: Pickup object
class Pickup: NSObject {
 
    var DriverId:Int?
    var EmployeeId:Int?
    var PickupId:Int?
    var PickupLineList = [PickupLine]()
    var SupplierId:Int?
    var WarehouseId:Int?
    var Address:Address?
    
}

File: SnackOverFlowSolution/DataObjects/Pickup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataObjects
{
    /// <summary>
    /// Robert Forbes
    /// 
    /// Created:
    /// 2017/04/13
    /// 
    /// DTO for Pickup
    /// </summary>
    public class Pickup
    {
        public List<PickupLine> PickupLineList { get; set; }
        public int? EmployeeId { get; set; }
        public int? DriverId { get; set; }
        public int? WarehouseId { get; set; }
        public int? SupplierId { get; set; }
        public int? PickupId { get; set; }

        public UserAddress address { get; set; }
    }
}
